### PR TITLE
LOC-1636 Detect Alpine linux and download specific binary

### PR DESCRIPTION
**Pull request by AdityaHirapara**

### CODE REVIEW

File: lib/LocalBinary.js [hunk 1]
```diff
@@ -16,12 +17,29 @@ function LocalBinary(){
...
+    try {
+      const stdout = childProcess.execSync('grep -w "NAME" /etc/os-release');
+      if (/Alpine/.test(stdout)) {
+        return true;
+      } else {
+        return false;
+      }
+    } catch(e) {
+      return false;
+    }
```
Comment (Raj-7799) [inline lib/LocalBinary.js:40]:
```suggestion
    try {
     return childProcess.execSync('grep -w "NAME" /etc/os-release').includes('Alpine');
    } catch(e) {
      return false;
    }
```

**Review by Raj-7799 — CHANGES_REQUESTED** [review]
Please add a test case for `LocalBinary()` and verify the httpPath

File: lib/LocalBinary.js [hunk 2]
```diff
@@ -16,12 +17,29 @@ function LocalBinary(){
     this.windows = true;
     this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal.exe';
   } else {
-    if(this.is64bits)
-      this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal-linux-x64';
-    else
-      this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal-linux-ia32';
+    if(this.isAlpine()) {
+      this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal-alpine';
```
Comment (akhil-browserstack) [inline lib/LocalBinary.js:21]: Screenshot 2020-08-27 at 7 18 59 PM
Alpine is not accessible here. I think there's a different path for that. Please check and add that.

Comment (AdityaHirapara) [reply lib/LocalBinary.js:21]: Its dummy for now, Bipul said he'll give link later.

**Review by Raj-7799 — APPROVED** [review]
LGTM

**Pull request merged**